src/components/Sidebar/index.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link
+} from "react-router-dom";
+
+import DashboardPanel from '../DashboardPanel';
+import RoutesPanel from '../RoutesPanel';
+import VolunteersPanel from '../VolunteersPanel';
+
+import NewRoutePanel from '../NewRoutePanel';
+
+import * as ROUTES from '../../constants/routes';
+
+import './index.css';
+
+const Sidebar = () => (
+  <div id="sidebar">
+
+    <div id="sidebar-menu">
+      <h2>Sidebar</h2>
+      <div id="sidebar-header-container">
+        <div id="sidebar-user-portrait">
+          (..)
+        </div>
+        <div id="sidebar-user-name">
+          John Doe
+        </div>
+      </div>
+
+      <ul>
+        <li><Link to={ROUTES.ADMIN_DASHBOARD}>Dashboard</Link></li>
+        <li><Link to={ROUTES.ADMIN_ROUTES}>Routes</Link></li>
+        <li><Link to={ROUTES.ADMIN_VOLUNTEERS}>Volunteers</Link></li>
+      </ul>
+
+      <div id="sidebar-adaptation">
+        <Switch>
```

Comment:
Seems to be a leftover div+switch, should it be removed?

Reply:
I added in an additional switch in case we want the actual sidebar UI to change depending on which route is active, so optional additional fields could live in the "sidebar-adaptation". We can remove it for now though.

Reply:
Leaving it is fine then.